src/wallet/wallet.cpp

```diff
@@ -686,7 +692,7 @@ bool CWallet::AccountMove(std::string strFrom, std::string strTo, CAmount nAmoun
     credit.nTime = nNow;
     credit.strOtherAccount = strFrom;
     credit.strComment = strComment;
-    AddAccountingEntry(credit, walletdb);
+    AddAccountingEntry(credit);
```

Comment:
Does this lose atomicity? Could duplicate the function to add a wrapper which does not require the walletdb pointer?